ristretto/ristretto.go

```diff
@@ -0,0 +1,93 @@
+package ristretto
+
+import (
+	"time"
+
+	"github.com/dgraph-io/ristretto"
+)
+
+// Storage interface that is implemented by storage providers.
+type Storage struct {
+	cache       *ristretto.Cache
+	defaultCost int64
+}
+
+// New creates a new storage.
+func New(config ...Config) *Storage {
+	cfg := configDefault(config...)
+	cache, err := ristretto.NewCache(&ristretto.Config{
+		NumCounters: cfg.NumCounters,
+		MaxCost:     cfg.MaxCost,
+		BufferItems: cfg.BufferItems,
+	})
+
+	if err != nil {
+		return nil
```

Comment:
We should send this error to the program somehow. With the other storages we panic, but returning `*Storage, error` and letting the programmer handle the error is probably better. But we should also be consistent :)

@ReneWerner87 what do you think?

Reply:
Exactly, good found

Reply:
I have added the panic line to provide consistency.